include/misc/exception.hpp

```diff
@@ -39,7 +39,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #else
 #   define EXCEPTION_HPP_NORETURN_MACRO [[noreturn]]
 #endif
-
+#if 0
```

Comment:
If we need this code nowhere, I suggest removing it. Otherwise, perhaps we need a more specific if than `0`?